Users/toml_config.rst

```diff
@@ -0,0 +1,308 @@
+Writing a coala Configuration File in TOML
+======================================================
+
+This document describes how to write configuration files for
+coala in TOML format.
```

Comment:
Do we have a list of recognized parameters a section can have such as `bears`, `ignore`, `files` etc. If we do, we should link it here with appropriate text.

Reply:
At line 60 I have added :
A list of settings required to write configuration can be found at
`Settings
<http://api.coala.io/en/latest/Developers/coala_settings.html>`